seq2science/rules/quantification.smk

```diff
@@ -261,6 +261,30 @@ elif config["quantifier"] == "kallistobus":
             {params.options} {input.reads} > {log} 2>&1
             """
 
+    rule kb_seurat_pp:
+        input:
+            expand([f"{{result_dir}}/{{quantifier}}/{custom_assembly(treps.loc[trep, 'assembly'])}-{trep}" for trep in treps.index], **config)
+        output:
+            html=f"{config['result_dir']}/seurat/{{quantifier}}/{{assembly}}/kb_seurat_pp.html",
+            qc_dir=directory(f"{config['result_dir']}/seurat/{{quantifier}}/{{assembly}}/qc")
+        priority: 1
+        conda:
+            "../envs/kb_scrna_R_seurat3.yaml"
+        params:
+            kb_dir=f"{config['result_dir']}/kallistobus"
+        resources:
+            R_scripts=1, # conda's R can have issues when starting multiple times
+        script:
+            f"{config['rule_dir']}/../scripts/kb_seurat_pp_rmd.R"
```

Comment:
Something we haven't discussed yet, but the docstring of rules gets auto-parsed by a little script we made to the docs: https://vanheeringen-lab.github.io/seq2science/content/all_rules.html, so adding a small explanation is nice :smile: 

Another thing is that we made a `explain` functionality, which gives a summary of what is done in each rule. Ideally this also gets added to this rule, see https://github.com/vanheeringen-lab/seq2science/blob/master/seq2science/rules/configuration_explain.smk, and any rule for a small example of how that gets invoked.

Reply:
Added rule explanation